app/Http/Controllers/ServerController.php

```diff
@@ -2,23 +2,42 @@
 
 namespace App\Http\Controllers;
 
+use Illuminate\Database\Eloquent\Model;
 use Illuminate\Http\Request;
 use App\Server;
 use App\Http\Controllers\Controller;
+use Illuminate\Support\Facades\App;
+use OpenGraph;
+use Twitter;
 
 class ServerController extends Controller {
 
     //http://regexr.com/3d8n1
-    const SERVER_REGEX = "/^(([\w-]+\.)?[\w-]+\.\w+|((2[0-5]{2}|1[0-9]{2}|[0-9]{1,2})\.){3}(2[0-5]{2}|1[0-9]{2}|[0-9]{1,2}))?$/";
 
     public function index() {
         $servers = Server::where('online', true)->orderBy("players", "desc")->paginate(5);
         return view('index', ['servers' => $servers]);
+
+
+        OpenGraph::addImage("favicon.ico"); // add image url
+        OpenGraph::setTitle("Minecraft-Database"); // define title
+        OpenGraph::setDescription("Database for Minecraft Servers");  // define description
+        OpenGraph::setUrl(url("/")); // define url
+        OpenGraph::setSiteName("Minecraft-Database");
+        OpenGraph::generate();
+
+
+        Twitter::setType("summary"); // type of twitter card tag
```

Comment:
Could you please the Twittercard use, because I think we could use OpenGraph for this.